### ISSUE TITLE
improve reliability of LoggingInputTest

### ISSUE THREAD

**Issue by ashtonhogan** [yegor256/cactoos#776]
I keep seeing this [error](https://ci.appveyor.com/project/yegor256/cactoos/build/1113) even though it has nothing to do with what I'm changing. I also don't have sufficient permission to trigger rebuild myself so this results in blocking PR's.

```
Tests run: 10, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.047 sec <<< FAILURE! - in org.cactoos.io.LoggingInputTest
logIntoCreatedLogger(org.cactoos.io.LoggingInputTest)  Time elapsed: 0.047 sec  <<< FAILURE!
java.lang.AssertionError: 
Expected: a string containing "Read 8 byte(s)"
     but: was ""
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
	at org.cactoos.io.LoggingInputTest.logIntoCreatedLogger(LoggingInputTest.java:231)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.apache.maven.surefire.junitcore.pc.Scheduler$1.run(Scheduler.java:393)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
```

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this issue

**Comment by 0crat** [comment]
@ashtonhogan[/z](https://www.0crat.com/u/ashtonhogan) this project will fix the problem faster if you donate a few dollars to it; just [click here](https://www.0crat.com/contrib/C63314D6Z) and pay via Stripe, it's very fast, convenient and appreciated; thanks a lot!

**Comment by llorllale** [comment]
@ashtonhogan I just ran all tests 5 times on a windows 8.1 box and can't reproduce this issue. What's the PR being blocked by this?

**Comment by llorllale** [comment]
@ashtonhogan ping

**Comment by Vatavuk** [comment]
@llorllale I had the same issue in PR #817  for ee45ae1. To resolve this I pushed a dummy commit just to restart tests. This happened to me in two PRs so far.

**Comment by Vatavuk** [comment]
@llorllale And third time now in #822

**Comment by llorllale** [comment]
@Vatavuk was it also happening to you in the appveyor build, or in travis?

**Comment by Vatavuk** [comment]
@llorllale only appveyor

**Comment by llorllale** [comment]
@Vatavuk yeah, just saw it [here](https://ci.appveyor.com/project/yegor256/cactoos/build/1220)

**Comment by llorllale** [comment]
@0crat in

**Comment by 0crat** [comment]
> @0crat in ([here](https://github.com/yegor256/cactoos/issues/776#issuecomment-386879520))

@llorllale Job [#776](https://github.com/yegor256/cactoos/issues/776) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
Bug was reported, see [§29](http://www.zerocracy.com/policy.html#29): +15 point(s) just awarded to @ashtonhogan[/z](https://www.0crat.com/u/ashtonhogan)

**Comment by 0crat** [comment]
The job [#776](https://github.com/yegor256/cactoos/issues/776) assigned to @victornoel[/z](https://www.0crat.com/u/victornoel), here is [why](https://www.0crat.com/footprint/C63314D6Z/1300646949); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html)

**Comment by 0crat** [comment]
@llorllale[/z](https://www.0crat.com/u/llorllale) please, pay attention to this issue

**Comment by 0crat** [comment]
@ashtonhogan[/z](https://www.0crat.com/u/ashtonhogan) this project will fix the problem faster if you donate a few dollars to it; just [click here](https://www.0crat.com/contrib/C63314D6Z) and pay via Stripe, it's very fast, convenient and appreciated; thanks a lot!

**Comment by ashtonhogan** [comment]
I don't know the steps to reproduce

**Comment by victornoel** [comment]
@ashtonhogan don't worry, I know the problem and the solution, PR is coming

**Comment by victornoel** [comment]
@0crat waiting for #855 to be merged

**Comment by 0crat** [comment]
> @0crat waiting for #855 to be merged ([here](https://github.com/yegor256/cactoos/issues/776#issuecomment-388011611))

@victornoel The impediment for [#776](https://github.com/yegor256/cactoos/issues/776) was registered successfully by @victornoel[/z](https://www.0crat.com/u/victornoel)

**Comment by 0crat** [comment]
@elenavolokhova[/z](https://www.0crat.com/u/elenavolokhova) please review this job completed by @victornoel[/z](https://www.0crat.com/u/victornoel), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#776](https://github.com/yegor256/cactoos/issues/776) is now out of scope

**Comment by elenavolokhova** [comment]
@ashtonhogan According to our Policy:
>Messages in a job always start with a name of a user they are addressed to;
[This comment](https://github.com/yegor256/cactoos/issues/776#issuecomment-387983679) was not addressed.

Please, confirm that you will address all your comments in future.

**Comment by ashtonhogan** [comment]
wtf

**Comment by victornoel** [comment]
@ashtonhogan this is part of QA job to ensure the process is followed correctly. Please confirm so that the job can be closed by 0crat and I can be paid :)

**Comment by victornoel** [comment]
@ashtonhogan lol, don't tell me, I was just explaining the process, tell @elenavolokhova ;)

**Comment by elenavolokhova** [comment]
@yegor256 I need your help here. Our Policy doesn't explain how to handle situation when someone directly rejects following our Policy rules. In this case violation is not critical and @victornoel should definitely be paid, but I can't tell that @ashtonhogan will respect our other process rules in future. Could you please advise?

**Comment by sleroy** [comment]
So funny.

**Comment by yegor256** [comment]
@elenavolokhova let's say the quality is "acceptable" and call it a day

**Comment by elenavolokhova** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @elenavolokhova[/z](https://www.0crat.com/u/elenavolokhova)
